lib/configure.js

```diff
@@ -272,6 +272,7 @@ function configure (gyp, argv, callback) {
       argv.push('-Dnode_gyp_dir=' + nodeGypDir)
       argv.push('-Dnode_lib_file=' + release.name + '.lib')
       argv.push('-Dmodule_root_dir=' + process.cwd())
+      argv.push('-Dnode_engine=' + (gyp.opts.node_engine || process.jsEngine || 'v8'))
```

Comment:
`process.jsEngine` doesn't look quite correct.  The current node binary might not be the one you're targeting.  (node -> node-webkit is a common example.)